### PR TITLE
feat(webvitals): More updates to Webvitals module components to use backend scores

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Updates more webvitals components to use backend scores:
- Updates more queries to use `performance_score(...)` and `weighted_performance_score(...)` functions
- Updates performance score breakdown chart to support using backend scores
- Reorganizes a few queries
- Update `calculatePerformanceScoreFromStored.tsx` to utilize properly `performance_score(...)` function
- Clean up some unused query fields
- Cleans up `orderby` from some queries, due to being replaced by sort hook

Backend score usage is still controlled by the `USE_STORED_SCORES` constant, which is hardcoded to false for the moment.

### CODE REVIEW

File: static/app/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsTimeseriesQuery.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+  result?.data?.['p75(measurements.lcp)']?.data.forEach((interval, index) => {
+    const lcp: number = result?.data?.['p75(measurements.lcp)']?.data[index][1][0].count;
+    const fcp: number = result?.data?.['p75(measurements.fcp)']?.data[index][1][0].count;
+    const cls: number = result?.data?.['p75(measurements.cls)']?.data[index][1][0].count;
+    const ttfb: number =
+      result?.data?.['p75(measurements.ttfb)']?.data[index][1][0].count;
+    const fid: number = result?.data?.['p75(measurements.fid)']?.data[index][1][0].count;
+    // This is kinda jank, but since events-stats zero fills, we need to assume that 0 values mean no data.
+    // 0 value for a webvital is low frequency, but not impossible. We may need to figure out a better way to handle this in the future.
```
Comment (gggritso) [inline static/app/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsTimeseriesQuery.tsx:105]: That's an awful lot of repetition! Iterating though `['lcp', 'fcp', 'cls', 'ttfb', 'fid']` would be way nicer

Comment (gggritso) [reply static/app/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsTimeseriesQuery.tsx:105]: Also applies to all the code below

File: static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored.tsx [hunk 2]
```diff
@@ -28,21 +28,12 @@ export function getWebVitalScores(data?: TableDataRow): ProjectScore {
...
-    ),
-    fidScore: Math.round(
-      getWebVitalScore(data, 'fid') * getWebVitalScoreWeight(data, 'fid') * 100
-    ),
+    lcpScore: Math.round(getWebVitalScore(data, 'lcp') * 100),
+    fcpScore: Math.round(getWebVitalScore(data, 'fcp') * 100),
+    clsScore: Math.round(getWebVitalScore(data, 'cls') * 100),
+    ttfbScore: Math.round(getWebVitalScore(data, 'ttfb') * 100),
+    fidScore: Math.round(getWebVitalScore(data, 'fid') * 100),
+    totalScore: Math.round(getTotalScore(data) * 100),
```
Comment (gggritso) [inline static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored.tsx:36]: These don't have to use weights?

File: static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useProjectWebVitalsScoresTimeseriesQuery.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+          index
+        ][1][0].count * 100;
+      const ttfb: number =
+        result?.data?.['weighted_performance_score(measurements.score.ttfb)']?.data[
+          index
+        ][1][0].count * 100;
+      const fid: number =
+        result?.data?.['weighted_performance_score(measurements.score.fid)']?.data[
+          index
+        ][1][0].count * 100;
```
Comment (gggritso) [inline static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useProjectWebVitalsScoresTimeseriesQuery.tsx:117]: Same note about repetition as before

**Review by gggritso — APPROVED** [review]
Makes sense! Nits only

File: static/app/views/starfish/components/chart.tsx [hunk 4]
```diff
@@ -86,6 +86,7 @@ type Props = {
   durationUnit?: number;
   errored?: boolean;
   forwardedRef?: RefObject<ReactEchartsRef>;
+  fullSeries?: boolean;
```
[the remark this answers is not in the document]
Comment (edwardgou-sentry) [reply static/app/views/starfish/components/chart.tsx]: 👍 renamed!

File: static/app/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsTimeseriesQuery.tsx (hunk 1 above)
Comment (edwardgou-sentry) [reply static/app/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsTimeseriesQuery.tsx]: I've cleaned this up to be much shorter now!

File: static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored.tsx (hunk 2 above)
Comment (edwardgou-sentry) [reply static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored.tsx:36]: This function is supposed to get you the individual 0 - 100 webvital scores (used to display in the tile components) and total score, so we don't need to apply weights here

File: static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useProjectWebVitalsScoresTimeseriesQuery.tsx (hunk 3 above)
Comment (edwardgou-sentry) [reply static/app/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useProjectWebVitalsScoresTimeseriesQuery.tsx]: Cleaned up!

**Pull request merged**